low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -110,124 +165,143 @@ def make_sspl_support_bundle_tarfile(parser, tmpFile):
             rmq_file.write("======================================================= \n")
         rmq_file.close()
 
-        if sspl_state == "active" and "-noencl" not in parser:
-            url = singleton_realstorencl.build_url("/")
-            COLLECTING_DEBUG_LOG_STARTED = False
-            for encl_trigger_log_retry_index in range(0, ENCL_TRIGGER_LOG_MAX_RETRY):
-                response = singleton_realstorencl.ws_request(url, singleton_realstorencl.ws.HTTP_POST, \
-                    post_data=f'/api/collectDebugData/"{SUPPORT_REQUESTOR_NAME}"{SUPPORT_EMAIL_ID}"{SUPPORT_CONTACT_NUMBER}')
+    def get_ipmi_sel_data(self):
+        ipmi_sel_cmd = f"ipmitool sel list > {self.ipmi_sel_data}"
+        subout = self._run_command(ipmi_sel_cmd)
+        if subout[2] != 0:
+            print("Required ipmitool is missing from the Node server to fetch sel data..!")
+        del subout
+
+    def get_consul_data(self):
+        self.consul_folder_name = f"consul_sspl_data-{str(int(time.time()))}"
+        consuldump = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/bin/consuldump.py -n {self.consul_folder_name} -l /tmp/ -d"
+        consul_sspl_cfg_cmd = f"{consuldump} consul_sspl_config sspl/"
+        consul_sspl_cache_cmd = f"{consuldump} consul_sspl_cache var/{PRODUCT_FAMILY}/sspl"
```

Comment:
lets add common config consul dump also as part of this collection
system_information
storage_enclosure
rabbitmq
and anything else sspl uses ..

Reply:
Done